kubemarine/admission.py

```diff
@@ -84,16 +84,20 @@ def enrich_inventory_pss(inventory: dict, _: KubernetesCluster) -> dict:
     for item in inventory["rbac"]["pss"]["defaults"]:
         if item.endswith("version"):
             verify_version(item, inventory["rbac"]["pss"]["defaults"][item], minor_version)
-    enabled_admissions = inventory["services"]["kubeadm"]["apiServer"]["extraArgs"].get("feature-gates")
+
     # add extraArgs to kube-apiserver config
-    if enabled_admissions:
```

Comment:
It's necessary to changes `kubernetes_admission_status` method in `check_paas.py` as well

Reply:
update check paas 
please check changes